.solhint.json

```
@@ -13,6 +13,13 @@
     "no-inline-assembly": "off",
     "no-empty-blocks": "off",
     "avoid-low-level-calls": "off",
-    "not-rely-on-time": "off"
+    "not-rely-on-time": "off",
+    "no-complex-fallback": "off",
+    "const-name-snakecase": "off",
+    "var-name-mixedcase": "off",
+    "func-name-mixedcase": "off",
+    "state-visibility": "off",
+    "reason-string": "off",
+    "no-global-import": "off"
```

```suggestion
    "no-complex-fallback": "off"
```
no need to turn off the others since we can follow that as good practice